content/docs/start/data-and-model-versioning.md

```diff
@@ -175,7 +175,7 @@ $ rm -f data/data.xml
 <tab title="Windows">
 
 ```
-> rmdir .dvc/cache
+> rmdir .dvc\cache
```

Comment:
I would keep $ for now. It's not an important detail to be honest.

Reply:
Done.